units/recalculate_change_points.go

```diff
@@ -63,69 +66,60 @@ func makeMessage(msg string, id model.TimeSeriesId) message.Fields {
 
 func (j *RecalculateChangePointsJob) Run(ctx context.Context) {
 	defer j.MarkComplete()
-	conf := model.NewCedarConfig(j.env)
-	err := conf.Find()
-	if err != nil {
-		grip.Error(message.WrapError(err, message.Fields{
-			"message": "Unable to get cedar configuration",
-		}))
-		return
+	if j.conf == nil {
+		j.conf = model.NewCedarConfig(j.env)
 	}
-	if conf.Flags.DisableSignalProcessing == true {
+	if j.conf.Flags.DisableSignalProcessing == true {
 		grip.Info(makeMessage("signal processing is disabled, skipping processing", j.TimeSeriesId))
 		return
 	}
 	if j.env == nil {
 		j.env = cedar.GetEnvironment()
 	}
-	db := j.env.GetDB()
-	detector := perf.NewMicroServiceChangeDetector(conf.ChangeDetector.URI, conf.ChangeDetector.User, conf.ChangeDetector.Token)
-	cur, err := model.GetTimeSeries(ctx, db, j.TimeSeriesId)
+	if j.ChangePointDetector == nil {
+		err := j.conf.Find()
+		if err != nil {
```

Comment:
don't log, just `j.AddError(errors.Wrap(err, "unable to get cedar configuration"))`